srcpkgs/ffmpeg/template

```diff
@@ -98,39 +76,47 @@ do_configure() {
 		_args+=" --extra-libs=-latomic"
 	fi
 
-	./configure --prefix=/usr --disable-debug --enable-gpl --enable-gnutls \
+	./configure \
+		--prefix='/usr' \
+		--extra-cflags='-I/opt/cuda/include' \
+		--extra-ldflags='-L/opt/cuda/lib64' \
```

Comment:
what's the purpose of this?

Reply:
switched to the branch i was using instead of another, not intended again sorry